src/framework/core/js/Fluid.js

```diff
@@ -13,7 +13,6 @@ Copyright 2007-2009 University of Toronto
 Copyright 2007-2009 University of California, Berkeley
 Copyright 2010-2011 Lucendo Development Ltd.
 Copyright 2010-2015 OCAD University
-Copyright 2011 Charly Molter
```

Comment:
Just to check: is this removal intentional?

Reply:
The code they contributed was removed by previous work from @the-t-in-rtf

Reply:
@amb26 should his Copyright be removed from FluidJSTests.js too?